demos/espressif/esp32_devkitc_esp_wrover_kit/common/application_code/espressif_code/mbedtls/port/include/sha1_alt.h

```diff
@@ -47,56 +47,6 @@ typedef struct
 }
 mbedtls_sha1_context;
 
```

Comment:
I realize that it has been like this, but it seems a shame to have 2 copies of the mbedtls/port folder (one for test and one for demos).  Do we anticipate these projects ever having differing files between the projects? Definitely not a TODO for this PR, but a question for future updates.

Reply:
Probably can be moved under `lib/third_party/mcu_vendor/espressif`, will take care in future PR.